packages/glimmer-wire-format/index.ts

```diff
@@ -78,7 +78,10 @@ export namespace Expressions {
   export const isHasBlockParams = is<HasBlockParams>('hasBlockParams');
 
   export function isValue(value: any): value is Value {
-    return value !== null && typeof value !== 'object';
+    if (value === null) {
+      return true;
```

Comment:
This confuses me. If `value` is `null` the `isValue` method is supposed to return true? Seems to me like null/undefined are **not** values?